dev/com.ibm.ws.security.javaeesec.cdi/resources/com/ibm/ws/security/javaeesec/cdi/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -17,50 +17,45 @@
-JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.explanation=The application deployment failed because both the login-config element and the HttpAuthenticationMechanism bean are defined. Only one of these authentication mechanisms can be used in your application.
-JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction=To enable the HttpAuthenticationMechanism bean, remove the login-config element from the web.xml file.
-
-JAVAEESEC_CDI_WARNING_NO_REALM_NAME=CWWKS1932W: The realm name for basic authentication is not set, so the default value, defaultRealm, was used.
-JAVAEESEC_CDI_WARNING_NO_REALM_NAME.explanation=Because realm name for the basic authentication is not defined, the realm name was set to the default value of defaultRealm.
-JAVAEESEC_CDI_WARNING_NO_REALM_NAME.useraction=To set the realm name, define it on a BasicAuthenticationMechanismDefinition annotation in the application. Alternatively, when the overrideHttpAuthenticationMechanism attribute is set as BASIC, set the value of attribute basicAuthenticationMechanismRealmName.
-
-JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM=CWWKS1933E: The value {0} of the configuration parameter {1} is invalid.
-JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.explanation=The value of the configuration parameter is not valid.
+JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM=CWWKS1933E: The password hash for the DatabaseIdentityStore attribute is not valid because the {1} configuration parameter has a value of {0} that is not valid.
+JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.explanation=The password hash for the DatabaseIdentityStore attribute is not valid because the value of the configuration parameter is not valid. <=== Explain what values are not valid, or given an example of what is not valid, or tell them what are common values that occur that are not valid, or tell them what is valid, etc.
 JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.useraction=Make sure that the configuration parameter is set correctly.
 
-JAVAEESEC_CDI_ERROR_PASSWORDHASH_BELOW_MINIMUM_PARAM=CWWKS1934E: The value {0} of the configuration parameter {1} is less than the minimum value {2}.
-JAVAEESEC_CDI_ERROR_PASSWORDHASH_BELOW_MINIMUM_PARAM.explanation=The value of the configuration parameter is less than the minimum value.
+JAVAEESEC_CDI_ERROR_PASSWORDHASH_BELOW_MINIMUM_PARAM=CWWKS1934E: The password hash for the DatabaseIdentityStore attribute is not valid because the {0} value of the {1} configuration parameter is less than the minimum value of {2}.
+JAVAEESEC_CDI_ERROR_PASSWORDHASH_BELOW_MINIMUM_PARAM.explanation=The value of the configuration parameter must be greater than or equal to the minimum value for the password hash of the DatabaseIdentityStore attribute to be valid.
 JAVAEESEC_CDI_ERROR_PASSWORDHASH_BELOW_MINIMUM_PARAM.useraction=Make sure that the configuration parameter value is equal to or greater than the minimum value.
```

Comment:
If you flip the clauses, the relationship between goal and actions is a little clearer to me. What do you think:

+JAVAEESEC_CDI_ERROR_PASSWORDHASH_BELOW_MINIMUM_PARAM.explanation=For the password hash of the DatabaseIdentityStore attribute to be valid, the value of the configuration parameter must be greater than or equal to the minimum value.

Reply:
I agree with you. Fixed.